swagger/swagger-listing.yaml

```diff
@@ -1186,11 +1186,17 @@
         PreferenceName:
           type: "string"
           description: "Preference name, (liveWork,COP,DCP,Neighborhood)"
+        totalUnits:
+          type: "integer"
+          description: "total number of units that are available for this preference"
+        totalApplications:
+          type: "integer"
+          description: "total number of applications that qualified for this preference"
         bucketResults:
```

Comment:
^^ these 6 lines are the proposed change, all the other line changes are just my text editor automatically removing extra whitespace